.circleci/maybe_deploy.clj

```diff
@@ -0,0 +1,32 @@
+(ns maybe-deploy
```

Comment:
maybe-deploy? Sound a bit confusing to me. Perhaps `deploy-release` or something like this?

Reply:
Renamed